fix(docs-infra): about page tab text not visible on small screens

ajitsinghkaler opened the pull request:
on small mobile in about page screens the top tab bar contains text which was not visible on small screens changed text size, margin and padding so that the text could be contained in these screens (320px to 480px)

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
Collaborartors not visible right on about page
![Angular - Angular Contributors](https://user-images.githubusercontent.com/39260684/79039618-1e64fb00-7c00-11ea-9c15-ea715da2c730.png)


Issue Number: N/A


## What is the new behavior?
Collaborators visible right on about page
![Angular - Angular Contributors (1)](https://user-images.githubusercontent.com/39260684/79039646-58360180-7c00-11ea-88f6-22eeda28b065.png)


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

mary-poppins commented:
You can preview 17d2fd2 at https://pr36576-17d2fd2.ngbuilds.io/.

gkalpak commented on aio/src/styles/2-modules/_contributor.scss:
```diff
@@ -19,6 +19,14 @@ aio-contributor-list {
   margin: 32px auto;
 
+  @media (max-width: 480px) {
+    .a.button.mat-button {
```
```suggestion
    a.button.mat-button {
```

Although, tbh, I would rather use `.filter-button` if possible, since the others are implementation details.

ajitsinghkaler replied:
we have to at least use .filter-button and .button to rewrite the styles

gkalpak replied:
@ajitsinghkaler, is it due to CSS specificity?

ajitsinghkaler replied:
yes the padding applied in _button .css is applied on the following classes a.button.mat-button

gkalpak replied:
I see, thx for clarifying. Let's leave it as is then. Fixing this properly would involve a more thorough rewrite of the styles.

gkalpak reviewed (approved):
Good catch :+1: Thx, @ajitsinghkaler!

One minor comment:
Please try to use proper sentences in the commit message to make it easier to read and more closely follow the [commit message guidelines][1] whenever possible. E.g.:

> fix(docs-infra): fix About page button text being truncated on small screens
>
> On small screens (e.g. on mobile), the text on some of the buttons in
> the "About" page was truncated. Changed the text size, margin and
> padding so that the the whole text is visible on such screens (320px to
> 480px).

[1]: https://github.com/angular/angular/blob/ca677481a2ac3675b84698e2d16fd379e1799e2c/CONTRIBUTING.md#commit

ajitsinghkaler committed:
fix(docs-infra): fix About page button text being truncated on small screens

On small screens (e.g. on mobile), the text on some of the buttons in
the About page was truncated. Changed the text size, margin and
padding so that the the whole text is visible on such screens (320px to
480px).

ajitsinghkaler commented:
@gkalpak made the changes please have a look

mary-poppins commented:
You can preview 697091e at https://pr36576-697091e.ngbuilds.io/.
